app/views/partials/user-settings-tabs.html

```diff
@@ -80,38 +80,56 @@
       <div class="components_div">
         <label style="float: left">
           Components
-          <i ng-show="uiSettings.showTooltips" tooltip-placement="top" tooltip="Select the components you wish to add to the visualizer" class="glyphicon glyphicon-question-sign tooltip-wide"></i>
+          <i ng-show="uiSettings.showTooltips" tooltip-placement="top" tooltip="Select the components you wish to add to the visualizer. Components are shown 'above the line'." class="glyphicon glyphicon-question-sign tooltip-wide"></i>
```

Comment:
I think we should change these referenced to 'above the line' to 'above the openhim-core' or 'below the openhim-core'. New users won't  understand this I don't think.